src/GitVersion.Core/VersionCalculation/IncrementStrategyFinder.cs

```diff
@@ -106,7 +106,11 @@ private IReadOnlyCollection<ICommit> GetCommitHistory(string? tagPrefix, Semanti
                 .ToHashSet()
         );
 
-        var intermediateCommits = GetIntermediateCommits(baseVersionSource, currentCommit, ignore).ToArray();
+        var intermediateCommits = this.repositoryStore.GetCommitLog(
+            baseVersionSource: baseVersionSource,
+            currentCommit: currentCommit,
+            ignore: ignore
```

Comment:
Does repeating the parameters contribute to code readability?

Reply:
I have changed it.